manager/notifier.go

```diff
@@ -168,13 +186,22 @@ func (n *notifier) sendPagerDutyNotification(serviceKey string, a *Alert) error
 	return nil
 }
 
-func (n *notifier) sendHipChatNotification(authToken string, roomId int32, color string, notify bool, a *Alert) error {
+func (n *notifier) sendHipChatNotification(op Op, config *pb.HipChatConfig, a *Alert) error {
 	// https://www.hipchat.com/docs/apiv2/method/send_room_notification
 	incidentKey := a.Fingerprint()
+	color := ""
+	status := ""
+	switch op {
```

Comment:
The PD notification implementation has a `default` error case. Should it be in the other ones too?